packages/paste-core/components/form/src/select/Select.tsx

```diff
@@ -49,7 +49,7 @@ const SelectElement = styled.select(
     fontWeight: 'fontWeightNormal',
     color: 'colorText',
     paddingTop: 'space30',
-    paddingRight: 'space40',
+    paddingRight: 'space100',
```

Comment:
Should this and the combobox styles match up here? I used `space120` and kept the `space40` right position above on combobox. Totally open to changing combobox. Just let me know and I'll create a ticket.

Reply:
Probably change Combobox. The intention was to make the gap between the chevron and the right border tighter. So then I matched (as close as I could) the whitespace between the left hand side of the chevron and the text.

You'd probably need a smaller token than 120 in that case too.

Reply:
Ticket created for this.